composer/trainer/trainer.py

```diff
@@ -1159,6 +1160,9 @@ def __init__(
             self.local_hparams = extract_hparams(locals())
             self.logger.log_hyperparameters(self.local_hparams)
 
+        # Log composer version.
+        self.logger.log_hyperparameters({'composer_version': composer_version})
```

Comment:
should be able to just use `composer.__version__`?

Reply:
but also might as well use the `get_composer_version` utility func in `collect_env.py`

Reply:
fixed